demos/python_demos/speech_recognition_demo/speech_recognition_demo.py

```diff
@@ -31,12 +32,13 @@ def build_argparser():
                         help="Beam width for beam search in CTC decoder (default 500)")
     parser.add_argument('-L', '--lm', type=str, metavar="FILENAME",
                         help="path to language model file (optional)")
-    parser.add_argument('-a', '--alphabet', type=str, metavar="FILENAME",
-                        help="path to alphabet file matching the model (defaults to the 28-symbol alphabet with English letters)")
-    parser.add_argument('--alpha', type=float, default=0.75, metavar="X",
-                        help="Language model weight (default 0.75)")
-    parser.add_argument('--beta', type=float, default=1.85, metavar="X",
-                        help="Word insertion bonus, ignored without LM (default 1.85)")
+
+    parser.add_argument('-p', '--profile', type=str, default='mds08x_en', metavar="NAME",
```

Comment:
```suggestion
    parser.add_argument('-p', '--profile', type=str, choices=PROFILES.keys(), metavar="NAME",
```
I would make the key be required so one wouldn't run mozilla-deepspeech-0.6.1 with default profile blindly (like I did at first :) ) So the help message should be updated as well.

Reply:
OK, made it a required argument. But without `choices` -- as it can accept a YAML file with profile. And added YAML profile example to the demo.